src/Utilities.jl

```diff
@@ -188,4 +188,26 @@ function time_to_seconds(s::String)
 end
 
 
+"""
+    integral(field)
+
+Return the integral for `field` along its spatial dimensions.
```

Comment:
Is the input always expected to be a 2D field? What will happen if a 3D field is passed in? Maybe also note explicitly that the return value will be a scalar

Reply:
I added a check to ensure that it is only 2d